ocs_ci/utility/csr.py

```
@@ -134,8 +135,113 @@ def get_nodes_csr():
+                continue
+
+            logger.warning(
+                f"{node_name} is running wrong RHCOS image: {current_image}. "
+                f"Rebasing to {correct_image}"
+            )
+            retcode, stdout, stderr = node.rpm_ostree_rebase(correct_image)
+            if retcode != 0:
+                logger.error(f"rpm-ostree rebase failed on {node_name}: {stderr}")
+                continue
+
+            logger.info(f"Rebooting {node_name} after successful rebase")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)